src/utils/url_utils.ts

```diff
@@ -42,8 +43,24 @@ export const generateServerUrl = (urlPath: string, baseUrl=API_BASE_URL, addTime
 	return _url.href;
 };
 
-export const generateWebUrl = (urlPath="", additionalParams: any=null,) => {
-	const url = `${WEB_APP_URL}${urlPath}`;
+export const generateApiUrl = (urlPath: string, addTimezone=false) => {
+	const apiBaseUrl = getSystemConfig().API_BASE_URL;
+	return generateBackendUrl(urlPath, apiBaseUrl, addTimezone);
+};
+
+export const generateApiHostUrl = (urlPath: string) => {
+	const baseUrl = getSystemConfig().API_HOST;
+	return generateBackendUrl(urlPath, baseUrl);
+};
```

Comment:
Host URL will be used for on prem customers?

Reply:
Will be used generally, its value will be different for onPrem users and will point to their resources